src/library/basetracktablemodel.cpp

```diff
@@ -741,6 +735,24 @@ QVariant BaseTrackTableModel::roleValue(
             // Not yet supported
             DEBUG_ASSERT(rawValue.isNull());
             break;
+        case ColumnCache::COLUMN_LIBRARYTABLE_COLOR:
```

Comment:
These 3 special cases could be pulled out of the main switch statement for Qt::ToolTipRole | kDataExportRole followed by M_FALLTHROUGH_INTENDED. Did you think about that?